<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hidato4x4</class>
 <widget class="QMainWindow" name="Hidato4x4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QTextEdit" name="text_number_12">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTextEdit" name="text_number_9">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="text_number_1">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QTextEdit" name="text_number_10">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QTextEdit" name="text_number_3">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="text_number_4">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="text_number_0">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="text_number_8">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QTextEdit" name="text_number_2">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QTextEdit" name="text_number_11">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QTextEdit" name="text_number_6">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QTextEdit" name="text_number_14">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QTextEdit" name="text_number_7">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QTextEdit" name="text_number_15">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QTextEdit" name="text_number_13">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTextEdit" name="text_number_5">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>871</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_new">
       <property name="text">
        <string>New Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_check">
       <property name="text">
        <string>Check solution</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
